spec/models/current_claim_spec.rb

```diff
@@ -89,7 +89,7 @@
       let(:ecp_eligibility) { build(:early_career_payments_eligibility, :eligible) }
       let(:lup_eligibility) { build(:levelling_up_premium_payments_eligibility, :eligible) }
 
-      let(:ecp_claim) { build(:claim, academic_year: "2022/2023", eligibility: ecp_eligibility) }
+      let(:ecp_claim) { build(:claim, academic_year: "2021/2022", eligibility: ecp_eligibility) }
```

Comment:
Is that year right?

Reply:
Yes, I added a comment to the factory as it doesn't work with the 2022/23 year.